clusterloader2/pkg/modifier/modifier.go

```diff
@@ -0,0 +1,45 @@
+package modifier
+
+import (
+	"k8s.io/perf-tests/clusterloader2/api"
+	"k8s.io/perf-tests/clusterloader2/pkg/flags"
+)
+
+type Modifier interface {
+	ChangeTest(*api.Config)
+}
+
+var (
+	skipSteps []string
+)
+
+func InitModifierFlags() {
```

Comment:
InitFlags (it's modifier package, we know these are Modifier ones)

Reply:
Done.